kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/internal/CertUtils.java

```diff
@@ -149,13 +149,19 @@ private static PrivateKey loadKey(InputStream keyInputStream, String clientKeyAl
   private static PrivateKey handleECKey(InputStream keyInputStream) {
     // Let's wrap the code to a callable inner class to avoid NoClassDef when loading this class.
     try {
+      if (Security.getProvider("BC") == null && Security.getProvider("BCFIPS") == null) {
+        new Callable<String>() {
```

Comment:
I am not sure to understand why you are creating a anonymous Callable object just to execute the call method without  reusing it afterwards?
Could probably be simplified with just 
```
Security.addProvider(new org.bouncycastle.jce.provider.BouncyCastleProvider());
```

Reply:
It's in case the dependency is not present at runtime.  There's a utility for this sort of thing called OptionalDependencyWrapper, but it's in common, not api.  We could consider moving that up to the api module for this.